packages/card/style.module.css

```diff
@@ -13,11 +13,15 @@
   position: relative;
   background-color: var(--background-color);
   transition: 0.2s ease;
-  transition-property: border-color, background-color;
+  transition-property: transform, border-color, box-shadow, background-color;
+  box-shadow: 0 16px 24px -16px #cbcbcd33, 0 8px 12px -8px #cbcbcd66;
```

Comment:
I think the intent based on Rays comment, is that this box-shadow is only applied in light appearance.

Reply:
Ah, yes that's a good point. Addressed in the latest commits